src/bcccp/tickets/season/SeasonTicketDAO.java

```diff
@@ -3,64 +3,108 @@
 import bcccp.tickets.season.ISeasonTicket;
 import bcccp.tickets.season.IUsageRecordFactory;
 
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Date;
+
 public class SeasonTicketDAO implements ISeasonTicketDAO {
```

Comment:
5. TYPE. Formatting: Google style Infringement
No javadocs present: "At the minimum, Javadoc is present for every public class,
and every public or protected member of such a class".